src/components/require-mfa.tsx

```diff
@@ -10,7 +10,7 @@ export const RequireMFA = () => {
     const router = useRouter()
 
     useLayoutEffect(() => {
-        if (user?.twoFactorEnabled === false && !pathname.startsWith('/account/mfa')) {
+        if (user?.twoFactorEnabled !== true && !pathname.startsWith('/account/mfa')) {
```

Comment:
I think this will redirect while clerk is loading, in that case user is null and there's a `isLoading` returned from useUser.  That's why we were previously testing that it was `false`.  https://clerk.com/docs/nextjs/hooks/use-user